reactive-blerpc-test/tests/end-to-end-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+/tmp/proto/protoc \
+  --plugin=protoc-gen-rx='generated/reactive-blerpc' \
+  --proto_path='.' \
+  -I 'blerpcproto/src/main/proto/' \
+  --rx_out='reactive-blerpc-test/tests' \
+  'reactive-blerpc-test/src/main/proto/test_service.proto'
+
+generated_service=reactive-blerpc-test/tests/com/device/proto/RxTestService.java
```

Comment:
Because unit tests for the generator don't actually test the template any more, we need to add all tests for template conditionals: deprecated classes/methods and the absence of javadocs for classes/methods. So, we need one more test with everything deprecated and docless.

Reply:
DONE